app/assets/javascripts/transactions.js

```diff
@@ -93,9 +97,39 @@ function show_hide_period_additional_input() {
   }
 }
 
-function datepicker_init(selector) {
+function addRemoveHints(fromCurr, toCurr) {
+  if (fromCurr != undefined && toCurr != undefined && fromCurr != toCurr ) {
+    var rate_hint = parseFloat(gon.current_org_rates[fromCurr + '_TO_' + toCurr]).toFixed(4);
+    if ($('.transfer_exchange_rate .help-block').html() != rate_hint) {
+      $('.transfer_exchange_rate .help-block').remove();
+      $('.transfer_exchange_rate').append('<span class="help-block">' + rate_hint + '</span>');
+    }
+
+    var amount = parseFloat($('#transfer_amount').val().replace(/\,/g,''));
+    var rate = parseFloat($('#transfer_exchange_rate').val().replace(/\,/g,''));
+    if (amount && rate) {
+      var end_sum = (amount * rate).toFixed(2);
+      if ($('.transfer_reference_id .help-block').html() != end_sum) {
+        $('.transfer_reference_id .help-block').remove();
+        $('.transfer_reference_id').append('<span class="help-block">' + end_sum + '</span>');
```

Comment:
@Gataniel move `$('.transfer_reference_id')` to variable like `$transferReference = $('.transfer_reference_id');`